front/src/routes/ApplicantsPage.svelte

```diff
@@ -46,7 +47,21 @@
     },
     { key: 'overflow', empty: true }
   ];
-  let rows = [];
+  interface CommitteeEntity {
+    id: number;
+    committee: CommitteeType;
+  }
+  interface RowEntity {
+    id: number;
+    year: string;
+    name: string;
+    resume: string;
+    email: string;
+    committees: CommitteeEntity[];
+    status: string;
+    committee_accepted: string;
+  }
```

Comment:
This is great but need to be moved to `interfaces.ts` file and then exported in this component